File: LDC.Atlas.Services.Execution/Application/Queries/Dto/InvoiceHomeSearch.cs
using LDC.Atlas.Services.Execution.Entities;
using System;

namespace LDC.Atlas.Services.Execution.Application.Queries.Dto
{
    public class InvoiceHomeSearch
    {
        public string InvoiceLabel { get; set; }

        public InvoiceType InvoiceType { get; set; }

        public DateTime InvoiceDate { get; set; }

        public string CounterpartyCode { get; set; }

        public DateTime DueDate { get; set; }

        public string VesselDescription { get; set; }

        public decimal TotalInvoiceValue { get; set; }

        public string Currency { get; set; }
    }
}

File: LDC.Atlas.WebApp/Client/app/shared/services/execution/dtos/invoice-home-search.ts
import { InvoiceTypes } from '../../../enums/invoice-type.enum';

export class InvoiceHomeSearch {
    public invoiceLabel: string;
    public invoiceType: InvoiceTypes;
    public invoiceDate: Date;
    public counterpartyCode: string;
    public dueDate: Date;
    public description: string;
    public counterpartyAdress: string;
    public totalInvoiceValue: number;
    public currency: string;
}
